assets/js/query_params.js

```diff
@@ -0,0 +1,32 @@
+import DOMPurify from 'dompurify';
```

Comment:
Curious the abuse vector we're looking to address with this sanitizer.

Should we only allowlist certain values? Or values that take a particular shape?